docs/guides/python_notebooks.md

```diff
@@ -0,0 +1,61 @@
+<!--
+Copyright 2025 Columnar Technologies Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Python Notebooks
+
+dbc can be installed and used directly in Python notebooks (such as Jupyter or Google Colab).
```

Comment:
Considering something like marimo, I wonder if it's appropriate to use the term "Python Notebook" to refer only to Jupyter / ipynb.
(IIUC, Colab is also Jupyter.)

So, I think this page should actually be called Jupyter instead of Python Notebook.

Reply:
Yep, I think a note like this would be sufficient:

> If you're using a Python notebook that doesn't support magics (`%`) or shell escapes (`!`) then use `subprocess.run` to run dbc commands from your Python code.